tests/sentry/api/endpoints/test_organization_metrics.py

```diff
@@ -463,7 +464,7 @@ def test_orderby(self):
             datasource="snuba",
             groupBy="transaction",
             orderBy="-count(sentry.transactions.measurements.lcp)",
-            limit=2,
+            per_page=2,
```

Comment:
It will need to be changed here: https://github.com/getsentry/sentry/blob/6d171b3213d29d1e3f00e9462a311d510e7878c7/static/app/utils/metrics/metricsRequest.tsx#L111
I can open frontend PR.